examples/official-storybook/stories/addon-actions.stories.js

```diff
@@ -97,4 +97,12 @@ storiesOf('Addons|Actions', module)
         Object (configured depth: 2)
       </Button>
     );
-  });
+  })
+  .add('Clearing the action logger', () => (
+    <div>
+      <p>Moving away from this story will clear the action logger</p>
```

Comment:
I think it would be more natural if it worked the other way around, and logger would be cleared when opening this story. Can we do this?

Reply:
I at first tried to do it the way @Hypnosphi described it, but it's slightly more difficult that way because the options are part of the actions and not the stories themselves. The way @ndelangen sounds more logical and should be of less complexity, so I will try my hands on that.